docs/http/http-metrics.md

```diff
@@ -2,13 +2,15 @@
 linkTitle: Metrics
 --->
 
+<!--- This file should not be edited. This file These was generated from YAML so changes need to be made
+to the source.  See refer to https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#yaml-to-markdown
+--->
```

Comment:
@reyang can you get something like this generated to help folk out who are conversant with your system?

Reply:
the file is not auto-generated, only the blocks between `<!-- semconv ... -->` and `<!-- endsemconv -->` are.

But I guess it's a valid concern that we can improve contributors life by pointing them at https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/README.md

Reply:
How about just before first occurrence of `<!-- semconv ... -->` add a comment that it's generated by YAML and point to https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#yaml-to-markdown

Or at the top of the file mention that the  `<!-- semconv ... -->`  blocks shouldn't be edited because they are generated by YAML?